src/main/java/com/example/dr_app/controller/ReservationController.java

```diff
@@ -0,0 +1,28 @@
+package com.example.dr_app.controller;
+
+import com.example.dr_app.model.Reservation;
+import com.example.dr_app.service.ReservationService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class ReservationController {
+
+    @Autowired
+    ReservationService reservationService;
+
+    @PostMapping("/makeReservation")
```

Comment:
Can you change the name to create-reservation ? Also please change the api endpoint names to kebab case?

Reply:
All the endpoints should be in kebab case?

Reply:
yes, sorry for late reply